templates/permission.tmpl

```diff
@@ -0,0 +1,75 @@
+{{define "title"}}Internal: Permission ({{.Permission.Name}}){{end}}
+{{define "content"}}
+    <div class="column is-10" style="min-height: 88vh">
+        <section class="hero is-info welcome is-small">
+            <div class="hero-body">
+                <div class="container">
+                    <h1 class="title">{{.Permission.Name}}</h1>
+                </div>
+            </div>
+        </section>
+        <br>
+        <div class="columns box" style="height: fit-content">
+            <div class="column is-2">
+                <div class="buttons" style="display: block">
+                    <a class="button is-warning is-outlined">
+                        Edit
+                    </a>
+                    {{if ne .Permission.Name "SuperUser"}}
+                        <a class="button is-danger is-outlined" onclick="deletePermissionModal()">
+                            Delete
+                        </a>
+                    {{end}}
+                </div>
+            </div>
+            <div class="column">
+                {{parsePermissionIntoHTML .Permission .UserPermissions}}
+            </div>
+        </div>
+    </div>
+{{end}}
+
+{{define "modals"}}
+    {{if ne .Permission.Name "SuperUser"}}
+        <div id="deletePermissionModal" class="modal">
+            <div class="modal-background"></div>
+            <div class="modal-content">
+                <div class="box">
+                    <article class="media">
+                        <div class="media-content">
+                            <div class="content">
+                                <p class="title">Are you sure you want to delete this permission?</p>
+                                <p><strong>This can be undone{{if eq .Permission.Name "SuperUser"}}<br>
```

Comment:
Did we decide we're forbidding deleting `SuperUser`? If so this seems redundant.

Reply:
I think so but missed that part